### DISCUSSION TITLE
[Feature request] Option to mirror your own/all player model(s)

### DISCUSSION THREAD

**Discussion by Ballistikk** [runelite/runelite#15307] [Ideas]
**Is your feature request related to a problem? Please describe.**
As ~10% of the population is left-handed (like myself), it's always a bit upsetting to see characters in games always being right-handed with no option to change it.

**Describe the solution you'd like**
The perfect solution to this would be to include an option in the GPU plugin to horizontally mirror your own and/all player models + animations to make it/them appear left-handed. Also, it might be necessary to disable this effect for certain items with writing on them, such as the 20th anniversary cape, to prevent them from being illegible.

**Describe alternatives you've considered**
You could technically achieve this much more easily by mirroring the whole game, minus UI elements, but it would make the whole map mirrored as well.

**Comment by Morgan-Demon** [comment]
Do you do everything in real life on the mirrored side?

**Reply by Ballistikk** [reply]
Well, yes, with my hands anyway. That's what being left-handed means...
